### ISSUE TITLE
[A11y] Add label for all non-test activities

### ISSUE THREAD

**Issue by rt4914** [oppia/oppia-android#3602]
[A11y] Add label for all non-test activities

[Reference](https://github.com/oppia/oppia-android/pull/3352/files/9c944564734be9aa46bf029cf14e6789ced95a7c#r672844473)

- [x] HomeActivity (Assigned to @vrajdesai78)
- [x] MyDownloadsActivity (Assigned to @bhaktideshmukh) 
- [x] ProfilePictureActivity
- [x] ProfileEditActivity(open)
- [x] SplashActivity (Assigned to @bkaur-bkj )
- [x] WalkthroughActivity (Assigned to @priyatanu)

Example PR: https://github.com/oppia/oppia-android/pull/3227/files

**Comment by rt4914** [comment]
@vrajdesai78 Can you work on this issue? If yes, suggest picking one of the mentioned files and creating PR for that first.

**Comment by vrajdesai78** [comment]
@rt4914 I want to work on this issue. I will create label for HomeActivity

**Comment by deepanshu731** [comment]
@rt4914 I want to create label for ProfilePictureActivity.

**Comment by vrajdesai78** [comment]
@rt4914 I am receiving static checks failed. Will I need to do anything else than adding label for HomeActivity in android manifest file and creating value in strings.xml

**Comment by rt4914** [comment]
@vrajdesai78 I have commented on PR

**Comment by bkaur-bkj** [comment]
@rt4914 sir I will make labels for SplashActivity pls assign that part to me.

**Comment by bkaur-bkj** [comment]
@rt4914 PTAL

**Comment by bkaur-bkj** [comment]
> @rt4914 sir PTAL

**Comment by priyatanu** [comment]
@rt4914 I want to work on this WalkthroughActivity activity

**Comment by rt4914** [comment]
@priyatanu Sure. Thanks.

**Comment by priyatanu** [comment]
> @priyatanu Sure. Thanks.

Hi @rt4914 , @BenHenning , kindly review my PR - https://github.com/oppia/oppia-android/pull/3983

**Comment by vivaansahai** [comment]
Hi, Can I work on `RecentlyPlayedActivity`?

**Comment by ankita240796** [comment]
Hi @rt4914, can I pick `MyDownloadsActivity`? Thanks!

**Comment by Rishavgupta12345** [comment]
can i work on RecentlyPlayedActivity

**Comment by priyatanu** [comment]
Hi @rt4914 , Can I work on Profile edit Activity?

**Comment by rt4914** [comment]
> can i work on RecentlyPlayedActivity

@Rishavgupta12345 I have actually removed `RecentlyPlayedActivity` from this list as it will require a bit different strategy and needs to finish some other issues first.

**Comment by Rishavgupta12345** [comment]
> Can I work on Profile edit Activity?

then Can I work on Profile edit Activity?

**Comment by Arjupta** [comment]
@Rishavgupta12345 have assigned ProfileEditActivity to you.

**Comment by bhaktideshmukh** [comment]
Hey! @ARJUPTA @rt4914 can you assign me MyDownloadsActivity?

**Comment by Arjupta** [comment]
@bhaktideshmukh assigned it to you.

**Comment by bhaktideshmukh** [comment]
Hey! @ARJUPTA Can you assign ProfileEditActivity to me as well.Thanks!!

**Comment by Rishavgupta12345** [comment]
> @Rishavgupta12345 have assigned ProfileEditActivity to you.

i was working on it why did you Un assigned me ??/

**Comment by Arjupta** [comment]
> > @Rishavgupta12345 have assigned ProfileEditActivity to you.
> 
> i was working on it why did you Un assigned me ??/

Dont worry @Rishavgupta12345 you are not unassigned, as you are working on a part of this issue hence assigning the whole issue to you will actually mean that you are working over all the sub issues. I removed you from assignees just to make it clear to other contributors.

@bhaktideshmukh you can see in the Issue description that `ProfileEditActivity` is currently assigned to @Rishavgupta12345 so I can't assign it to you.

**Comment by bhaktideshmukh** [comment]
> > > @Rishavgupta12345 have assigned ProfileEditActivity to you.
> > 
> > 
> > i was working on it why did you Un assigned me ??/
> 
> Dont worry @Rishavgupta12345 you are not unassigned, as you are working on a part of this issue hence assigning the whole issue to you will actually mean that you are working over all the sub issues. I removed you from assignees just to make it clear to other contributors.
> 
> @bhaktideshmukh you can see in the Issue description that `ProfileEditActivity` is currently assigned to @Rishavgupta12345 so I can't assign it to you.

Sure no problem:)

**Comment by shankarpriyank** [comment]
Hi @bkaur-bkj I would like to work on ProfileEditActivity issue, can you please assign this to me

**Comment by Arjupta** [comment]
> Hi @bkaur-bkj I would like to work on ProfileEditActivity issue, can you please assign this to me

@Rishavgupta12345 if you are unable to work on this issue so let us know. Its has been 12 days since it was assigned to you.

**Comment by Rishavgupta12345** [comment]
> > Hi @bkaur-bkj I would like to work on ProfileEditActivity issue, can you please assign this to me
> 
> @Rishavgupta12345 if you are unable to work on this issue so let us know. Its has been 12 days since it was assigned to you.

I have completed this issue but since my one pull request is pending on review:- #4088  , I am not claiming another pull request . AS SOON AS my previous pull request gets merged I will make the pull request for this issue

AND MOREOVER you are the one who is assigned to review my pull request of #4088

**Comment by Rishavgupta12345** [comment]
> > > Hi @bkaur-bkj I would like to work on ProfileEditActivity issue, can you please assign this to me
> > 
> > 
> > @Rishavgupta12345 if you are unable to work on this issue so let us know. Its has been 12 days since it was assigned to you.
> 
> I have completed this issue but since my one pull request is pending on review:- #4088 , I am not claiming another pull request . AS SOON AS my previous pull request gets merged I will make the pull request for this issue

AND MOREOVER, you are the one assigned to review my pull request of #4088.

**Comment by Arjupta** [comment]
> > > Hi @bkaur-bkj I would like to work on ProfileEditActivity issue, can you please assign this to me
> > 
> > 
> > @Rishavgupta12345 if you are unable to work on this issue so let us know. Its has been 12 days since it was assigned to you.
> 
> I have completed this issue but since my one pull request is pending on review:- #4088 , I am not claiming another pull request . AS SOON AS my previous pull request gets merged I will make the pull request for this issue
> 
> AND MOREOVER you are the one who is assigned to review my pull request of #4088

I understand that you are waiting for your first PR to merge, but if that's the case then you shouldn't ask for more issues to work on. There are plenty of issues that you can pick up after your PR gets merged. As you have already not made any commits regarding this issue so I am unassigning you, you can ask me for any other issue once you are ready. 

About the PR review, I have suggested changes and didn't recieve any replies or changes from your end. Neither me nor Rajat is able to see your replies. PTAL

**Comment by bkaur-bkj** [comment]
@ARJUPTA  pls do assign Profile edit activity Issue to @shankarpriyank

**Comment by BenHenning** [comment]
Nice to see this cleanup get fully resolved. :)

**Comment by rt4914** [comment]
Opening this issue as one thing is pending: https://github.com/oppia/oppia-android/pull/4126

**Comment by Akshatkamboj14** [comment]
can you assign me this?

**Comment by deonwaju** [comment]
I would like to add the label for profileEditActivity

**Comment by MohitGupta121** [comment]
@DeonWaju assigned go ahead.
If you have any query feel free to ask me.

**Comment by deonwaju** [comment]
Thank you.

**Comment by chrislee115** [comment]
Hi! Can I work on ProfileEditActivity?

**Comment by adhiamboperes** [comment]
> Hi! Can I work on ProfileEditActivity?

Hi @chrislee115, someone is already working on this issue, but you can look at this other one: #3557  or #2116

**Comment by adhiamboperes** [comment]
@BenHenning I'm closing this issue since the last task has been completed.
